lib/crowi/index.ts

```diff
@@ -134,7 +168,7 @@ class Crowi {
 
   // getter/setter of model instance
   //
-  model(name, model) {
+  model(name: any, model?: any) {
```

Comment:
I think we can change to `string` and `number`.(or only string)

Reply:
fixed @ https://github.com/crowi/crowi/pull/486/commits/da2644158198183c41be09c25e6d25efdef7dd49